### ISSUE TITLE
Client doesn't run anymore

### ISSUE THREAD

**Issue by straylight-orbit** [cloudhead/nakamoto#113]
It permanently gets stuck on

```
[INFO  nakamoto_client::client] Filters height = 0
[INFO  nakamoto_client::client] Loading filter headers from store..
[INFO  nakamoto_client::client] Verifying filter headers..
```

Clearing the data directory doesn't help. Revision `e74e83d6271f063d53b1e67231c17819d8800649`.

**Comment by cloudhead** [comment]
Yeah, I think it's related to https://github.com/cloudhead/nakamoto/issues/111

For now it should work if you subscribe to the loading events:

```rust
let h = client.handle();
for _event in h.loading() {}
...
thread::spawn(|| client.run())
```

**Comment by cloudhead** [comment]
Potential fix in https://github.com/cloudhead/nakamoto/pull/114

**Comment by cloudhead** [comment]
Closed with #114
